server/swagger.json

```diff
@@ -4080,6 +4087,22 @@
 				}
 			}
 		},
+    "DashboardRange": {
+      "type": "object",
+      "description": "A description of a particular axis for a visualization",
+      "properties": {
+        "bounds": {
+          "type": "array",
+          "minItems": 0,
+          "maxItems": 2,
+          "description": "The extents of an axis in the form [lower, upper]. Clients determine whether bounds are to be inclusive or exclusive of their limits",
+          "items": {
+            "type": "integer",
+            "format": "int32"
```

Comment:
I matched this to the dimensions of the cell thinking that... well, they should match. However thinking about this more, you're right. If I'm recording stats with very high magnitudes, I'd still want to visualize them without doing math to compress them into the lower 32-bits. Will change.